File: trunk/src/com/akrog/tolomet/WindProviderType.java
package com.akrog.tolomet;

public enum WindProviderType {
	Euskalmet(0), MeteoNavarra(1), Aemet(2);
	
	private final int value;
	
	private WindProviderType(int value) {
        this.value = value;
    }
	
	public int getValue() {
        return value;
    }
}

File: Project/tolometShared/src/commonMain/kotlin/com/akrog/tolomet/providers/WindProviderType.kt
package com.akrog.tolomet.providers

enum class WindProviderType(
    val code: String,
    val quality: WindProviderQuality,
    val isDynamic: Boolean
) {
    Euskalmet("EU", WindProviderQuality.Good, true),
    Aemet("AE", WindProviderQuality.Poor, true),
    Noromet("NO", WindProviderQuality.Good, true);
    // Add others as needed for proof of concept
}
